### PR TITLE
[WIP] Add states types and remove useless variables and functions

### PR DESCRIPTION
**Pull request by lcordier42**
Is it possible to know the types of channel and chatClient that come from twilio modules?
All the functions are public at the moment.

### CODE REVIEW

File: components/ChatApp.tsx [hunk 1]
```diff
@@ -6,217 +6,45 @@ import { NameBox } from "./NameBox";
...
             newChannel: "",
             newMessage: "",
             offlineMembers: [],
             onlineMembers: [],
             token: "",
         };
-        this.name = "";
-    }
-
-    public render() {
```
Comment (brikou) [inline components/ChatApp.tsx:46]: Why do you remove all?

Comment (lcordier42) [reply components/ChatApp.tsx]: render is at the end of file

Comment (lcordier42) [reply components/ChatApp.tsx:46]: Sorry, I did not see my message was waiting.
I only remove what was useless, chatReady was unused for example, and this.name is declared further

Comment (brikou) [reply components/ChatApp.tsx:46]: ping @lcordier42

Comment (lcordier42) [reply components/ChatApp.tsx:46]: ping @brikou

**Pull request merged**